content/en/docs/languages/java/configuration.md

```diff
@@ -180,8 +180,8 @@ exporters.
-| `otel.exporter.otlp.endpoint`                              | The OTLP traces, metrics, and logs endpoint to connect to. Must be a URL with a scheme of either `http` or `https` based on the use of TLS. If protocol is `http/protobuf` the version and signal will be appended to the path (e.g. `v1/traces`, `v1/metrics`, or `v1/logs`).                                                                                                                                                       | `http://localhost:4317` when protocol is `grpc`, and `http://localhost:4318/v1/{signal}` when protocol is `http/protobuf`. |
-| `otel.exporter.otlp.{signal}.endpoint`                     | The OTLP {signal} endpoint to connect to. Must be a URL with a scheme of either `http` or `https` based on the use of TLS.                                                                                                                                                                                                                                                                                                           | `http://localhost:4317` when protocol is `grpc`, and `http://localhost:4318/v1/{signal}` when protocol is `http/protobuf`. |
+| `otel.exporter.otlp.endpoint`                              | The endpoint to send all OTLP traces, metrics, and logs to, commonly the address of a collector. Must be a URL with a scheme of either `http` or `https` based on the use of TLS. If protocol is `http/protobuf` the version and signal will be appended to the path (e.g. `v1/traces`, `v1/metrics`, or `v1/logs`).                                                                                                                 | `http://localhost:4317` when protocol is `grpc`, and `http://localhost:4318/v1/{signal}` when protocol is `http/protobuf`. |
+| `otel.exporter.otlp.{signal}.endpoint`                     | The endpoint to send OTLP {signal} to, commonly the address of a collector. Must be a URL with a scheme of either `http` or `https` based on the use of TLS.                                                                                                                                                                                                                                                                         | `http://localhost:4317` when protocol is `grpc`, and `http://localhost:4318/v1/{signal}` when protocol is `http/protobuf`. |
```

Comment:
It could also be any other kind of OTLP ingestion layer, so I am not sure if "commonly" is not too strong? maybe "often" or "for example", also I would recommend that in that case we say "address of an OpenTelemetry Collector" to be specific